src/browser/index.js

```diff
@@ -42,17 +43,40 @@ if (isReload) {
 document.body.classList.toggle('use-native-titlebar',
                                Runtime.useNativeTitlebar());
 
+const replay =
+  () => {
+    const request = new XMLHttpRequest();
+    // We use sync XHR as we have nothing else to render otherwise.
```

Comment:
That works with Servo, but in Gecko, this still spins the event loop (yeah, gecko does that with sync xhr and alerts).  Not sure how that affects the rest of our code, but this doesn't block the event loop.